frontend/javascripts/libs/utils.ts

```diff
@@ -1300,3 +1300,10 @@ export function generateRandomId(length: number) {
   }
   return result;
 }
+
+export function getPhraseFromCamelCaseString(stringInCamelCase: string): string {
```

Comment:
I'm not a big fan of such automatic conversions to make something readable for humans. I'd rather have explicit (and type-enforced) mappings. we can keep it for this iteration, but just fyi :)

Reply:
Alright! I added it to the todo-list for future iterations